packages/desktop/package.json

```diff
@@ -43,7 +43,7 @@
         "copy-webpack-plugin": "^7.0.0",
         "cross-env": "^7.0.2",
         "css-loader": "^5.0.1",
-        "electron": "22.0.0",
+        "electron": "23.1.0",
```

Comment:
Let's upgrade it to the latest 22 version, because otherwise we'll have windows support issues.